website/content/en/docs/olm-integration/quickstart-bundle.md

```diff
@@ -111,25 +112,52 @@ Now we're ready to test and deploy the Operator with OLM.
 
 ### Testing bundles
 
-<!-- TODO(jmrodri): `run bundle` usage here -->
+Before proceeding, make sure you've [Installed OLM](#enabling-olm) onto your
+cluster. First, we need to build our bundle. To build a memcached-operator
```

Comment:
Since the prereq is important it should probably float
```suggestion
Before proceeding, make sure you've [Installed OLM](#enabling-olm) onto your cluster.

First, we need to build our bundle. To build a memcached-operator
```